mitodl/ocw-hugo-themes
# pass page context to video_expandable_tab wherever it's used

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1137

#### What's this PR do?
In https://github.com/mitodl/ocw-hugo-themes/pull/1123, we updated the `video_expandable_tab.html` partial to use `.RenderString` as opposed to `markdownify`. Since `.RenderString` is a property of the `Page` context, the page context has to be passed into the partial for it to work. This PR passes that context on the remaining references to `video_expandable_tab.html` that were not handeld in https://github.com/mitodl/ocw-hugo-themes/pull/1123. A new playwright test has also been added to ensure that the expected tab titles and contents are rendered where expected. Playwright was upgraded to a new version as part of this to take advantage of the `Locator.all()` function.

#### How should this be manually tested?
 - Download https://github.mit.edu/ocw-content-rc/15.s21-january-iap-2014 and set your `COURSE_CONTENT_PATH` and `OCW_TEST_COURSE` env vars to point to it
 - Start the course with `yarn start course`
 - Ensure that at http://localhost:3000/resources/part-2-legal-issues/, a tab with the title "Lecture Notes" is rendered, and inside of it is a link to a PDF resource. This link will not work locally, because it's a hard coded markdown link with the `/courses` prefix, but it will work in production. Converting links in the `optional_text` field to resource link shortcodes in `ocw-studio` is a separate issue.
